src/core/reactor.cc

```diff
@@ -1905,6 +1905,9 @@ reactor::open_file_dma(std::string_view nameref, open_flags flags, file_open_opt
                     attr.fsx_extsize = std::min(options.extent_allocation_size_hint,
                                         file_open_options::max_extent_allocation_size_hint);
 
+                    constexpr uint32_t min_extent_size_hint_alignment{128u << 10}; // 128KB
```

Comment:
Please, put it next to `file_open_options::max_extent_allocation_size_hint` with a comment that XFS rejects hints that are not aligned, so we try to please it with the best guess

Reply:
Sure, done